ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/MessageScrubber.java

```diff
@@ -87,6 +90,19 @@
       .put("/submit/sslreports/", "1585144") //
       .build();
 
+  // Static list prepared for bug 1751753, pulled from
+  // https://searchfox.org/mozilla-central/source/services/settings/dumps/main/search-telemetry-v2.json
+  // TODO: Pull list from RemoteSettings.
```

Comment:
This link 404s for me.

I think in future versions of this we'll want to make this an artifact or similar, as we don't have outbound access from the decoder and because it's generally safer to rely on e.g. gcs.

Reply:
I'll need to better understand details of how often this collection would be changing and whether we'd be at risk of a race condition where the pipeline might sanitize legitimate search codes due to only updating the list on pipeline restart.

Reply:
> This link 404s for me.

The change with this new file is not merged yet.